docs/v3/source/includes/upgrade_guide/conceptual_changes/_resource_summaries.md

```diff
@@ -21,10 +21,42 @@ filter by the parent resource. See below for examples of summaries in V3.
 
 - To fetch all apps in a space, use `GET /v3/apps?space_guids=<space-guid>`.
   Passing `include=space` will include the space resource in the response body.
+- To fetch all service offerings in a space use `GET
+  /v3/service_offerings?space_guids=<space-guid>`. 
 - To fetch all service instances in a space use `GET
   /v3/service_instances?space_guids=<space-guid>`. Use the
```

Comment:
The `fields` parameter is available for service offerings as well. I wonder if we should repeat that sentence in both or point it out after this list.